packages/schemas/alterations/next-1678284778-restrict-internal-roles.ts

```diff
@@ -0,0 +1,62 @@
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      update roles
+      set name = '#internal:admin', description = 'Internal admin role for Logto tenant ' || tenant_id || '.'
+      where name = 'admin'
+      and tenant_id != 'admin';
+
+      update roles
+      set description = 'Admin tenant admin role for Logto tenant ' || substring(name from 0 for strpos(name, ':admin')) || '.'
+      where name like '%:admin'
```

Comment:
Wondering whether user could have their own `%:admin` liked role defined in their DB e.g. `host:admin`

Reply:
yes they can. this convention is for admin tenant only